### PR TITLE
ESLint implementation and fixes

### PR DESCRIPTION
**Pull request by mimiflynn**
Addresses issue #85

### CODE REVIEW

File: scripts/googleapi2events.js
Comment (mimiflynn) [inline scripts/googleapi2events.js:1]: This file isn't used in the application, so I moved it to a `scripts/` directory for utilities like this.

File: src/App.jsx
Comment (mimiflynn) [inline src/App.jsx:1]: Our files has a mix of spaces and tabs, so the eslint will automatically fix this.

**Comment by psmulovics** [comment]
@maoo , @TheJuanAndOnly99 , I think the whole ics2events file could be removed, and if we do that, we could remove the rrule dependency completely?

**Comment by maoo** [comment]
> @maoo , @TheJuanAndOnly99 , I think the whole ics2events file could be removed, and if we do that, we could remove the rrule dependency completely?

👍  from me. I created an `ics2events` branch so we keep a copy of the code, just in case.

**Review by psmulovics — APPROVED** [review]

**Pull request merged**